addons/base_vat/tests/test_validate_ruc.py

```diff
@@ -106,6 +106,24 @@ def test_nif_de(self):
         with self.assertRaises(ValidationError):
             test_partner.write({'vat': "136695978"})
 
+    def test_rut_uy(self):
+        test_partner = self.env["res.partner"].create({"name": "UY Company", "country_id": self.env.ref("base.uy").id})
+        # Set a valid Number
+        test_partner.write({"vat": "215521750017"})
```

Comment:
```suggestion
        test_partner.vat = "215521750017"
```
[nit] Doesn't make much sense to use the write notation for a single field, I prefer the syntactic sugar.